### PR TITLE
Split manual for RustRover and IDEA in Getting Started/External Linter

### PR DESCRIPTION
**Pull request by kochaika**
Based on the suggestion from https://github.com/jetbrains-academy/rustlings-course/pull/300

### CODE REVIEW

File: Introduction/Getting started/External Linter/task.md [hunk 1]
```diff
@@ -1,13 +1,21 @@
-## Enable External Linter
+IntelliJ Rust plugin doesn't detect all the errors. It relies on the Rust compiler to do that. While you are learning Rust, it is useful to see errors as you type. To have this behavior, we recommend to enable an external linter as follows.
```
Comment (stephen-hero) [inline Introduction/Getting started/External Linter/task.md:1]: The IntelliJ Rust plugin doesn't detect all errors on its own; it relies on the Rust compiler for that. While learning Rust, it is helpful to see errors as you type. To achieve this, we recommend enabling an external linter as follows.

File: Introduction/Getting started/External Linter/task.md [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
 
-IntelliJ Rust plugin doesn't detect all the errors. It relies on the Rust compiler to do that. While you are learning Rust, it is useful to see errors as you type. To have this behavior, we recommend to enable an external linter as follows:
+Once you do that, %IDE_NAME% will report all the errors IntelliJ Rust plugin or Rust compiler detect.
```
Comment (stephen-hero) [inline Introduction/Getting started/External Linter/task.md:3]: Once you do that, %IDE_NAME% will report all errors detected by either the IntelliJ Rust plugin or the Rust compiler.

File: Introduction/Getting started/External Linter/task.md [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
 
-1. Go to **Settings / Preferences | Languages & Frameworks | Rust | External Linters** (or directly **Settings / Preferences | Rust | External Linters** if opened in RustRover).
-2. Set the parameters as follows: 
-    - Select **Cargo Check** in the **External Tool:** list;
-    - Check the **Run external linter to analyze code on the fly** checkbox.
+Depending on the IDE you are using, the flow may be slightly different.
```
Comment (stephen-hero) [inline Introduction/Getting started/External Linter/task.md:5]: The steps may vary slightly depending on the IDE you're using.

File: Introduction/Getting started/External Linter/task.md [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
 
-![External Linters](images/external-linters.png)
+### RustRover
+1. Go to **Settings / Preferences | Rust | External Linters**.
+2. Set the parameters as follows:
+   - Check the **Run external linter on the fly** checkbox.
```
Comment (stephen-hero) [inline Introduction/Getting started/External Linter/task.md:10]: Check the **Run external linter on the fly** box.

File: Introduction/Getting started/External Linter/task.md [hunk 5]
```diff
@@ -1,13 +1,21 @@
...
+   - Select **Cargo Check** in the **External Tool:** list;
+![External Linters](images/rustrover_external_linters.png)
 3. Press **OK**.
 
-Once you do that, %IDE_NAME% will report all the errors IntelliJ Rust plugin or Rust compiler detect.
+### IntelliJ IDEA
+1. Go to **Settings / Preferences | Languages & Frameworks | Rust | External Linters**.
+2. Set the parameters as follows:
+   - Check the **Run external linter on the fly** checkbox.
```
Comment (stephen-hero) [inline Introduction/Getting started/External Linter/task.md:18]: Check the **Run external linter on the fly** box.

**Review by stephen-hero — CHANGES_REQUESTED** [review]

**Review by stephen-hero — APPROVED** [review]

**Review by Geravant — APPROVED** [review]

**Pull request merged**